base/visualization/R/visually.weighted.watercolor.plots.R

```diff
@@ -210,7 +210,7 @@ vwReg <- function(formula, data, title = "", B = 1000, shade = TRUE, shade.alpha
       }
       
       gg.poly <- list(
-        ggplot2::geom_polygon(data = d3, ggplot2::aes(x = x, y = value, color = NULL, fill = col, group = group)),
+        ggplot2::geom_polygon(data = d3, ggplot2::aes(x = d3$x, y = d3$value, color = NULL, fill = col, group = d3$group)),
```

Comment:
Not sure if the `d3$` prefix is needed for the `col` variable as well. 
( There is actually a `col` variable defined in the function )